recipes/geospatial/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "geospatial" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/geospatial-{{ version }}.tar.gz
+  sha256: 93edf988a6606e21f8b13251c2e8c6c073ef5b8f7b96d51ec38ef8b0a00e4587
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - geemap
+    - leafmap
+    - python >=3.7
```

Comment:
```suggestion
    - python >=3.5
```

Reply:
I have updated the upstream to require Python >=3.7